{{cookiecutter.project_slug}}/_/deployment/python/deployment/application/base/kustomization.yaml

```diff
@@ -5,14 +5,18 @@ commonLabels:
   app: {{ cookiecutter.project_slug }}
 {%- endif %}
   component: application
+configMapGenerator:
+- name: application
+  envs:
+  - application.env
 crds:
 - route-crd.yaml
 {%- if cookiecutter.environment_strategy == 'shared' %}
 namespace: {{ cookiecutter.project_slug }}
 {%- endif %}
 resources:
-- deployment.yaml
-- route.yaml
-- secret.yaml
-- service.yaml
-- rolebinding.yaml
+  - deployment.yaml
+  - route.yaml
+  - secret.yaml
+  - service.yaml
+  - rolebinding.yaml
```

Comment:
The change of the indentation is probably unnecessary. I believe, we try to keep the indentation to the minimum needed in YAML.